it/src/test/java/com/linecorp/armeria/server/grpc/interop/ArmeriaGrpcServerInteropTest.java

```diff
@@ -77,12 +85,17 @@
  */
 public class ArmeriaGrpcServerInteropTest extends AbstractInteropTest {
 
+    @Rule
+    public TestRule timeout = new DisableOnDebug(Timeout.millis(Long.MAX_VALUE));
```

Comment:
This magic is supposed to replace the test's specified timeout with this one only when debugging. It didn't work though... (maybe since tests are in a superclass) and I didn't mean to leave it in there either way, removed.